tools/clang/test/SemaHLSL/array-index-out-of-bounds-HV-2016.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -Wno-unused-value -fsyntax-only -ffreestanding -HV 2016 -verify -verify-ignore-unexpected=note %s
+// RUN: %dxc -Tlib_6_3 -Wno-unused-value -HV 2016 -verify -verify-ignore-unexpected=note %s
```

Comment:
This test was already using `-verify-ignore-unexpected=note`. What was that doing before this change?

Reply:
It doesn't go thru dxc before.